### DISCUSSION TITLE
isssue in Browser Bookmarks

### DISCUSSION THREAD

**Discussion by logann99** [Flow-Launcher/Flow.Launcher#1618] [Show and tell]
Browser Bookmarks its not working for me in V1.9.5 and 1.9.4
![image](https://user-images.githubusercontent.com/101401329/206919079-276ac8d4-7478-4cfd-8d00-9ca345df29f6.png)

**Comment by logann99** [comment]
i use brave browser

**Comment by jjw24** [comment]
Closing and please go to https://github.com/Flow-Launcher/Flow.Launcher/issues/2384
